R/helper.R

```diff
@@ -36,3 +36,17 @@ dataset_in_dir <- function(dir = rappdirs::user_data_dir()) {
   }
   data_exist
 }
+
+
+ask_yn <- function(...) {
```

Comment:
Add some roxygen for this:

```
#' @importFrom crayon green
#' @importFrom utils menu
```

Reply:
You'll also need to run:

```
usethis::use_package("crayon")
usethis::use_package("utils")
```

Or just add crayon and utils to the Imports section of the DESCRIPTION file manually.